docs/swagger.yml

```diff
@@ -203,6 +203,29 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/SubstituteIds'
+  /users/{user_id}/organizations:
+    get:
+      tags:
+        - Users
+      summary: Get substituted organizations
+      description: Get list of organizations that you can create invoices on behalf of
```

Comment:
`Get list of IČOs that requested user can create invoices on behalf of`
So it is clear this list is related to user from URL rather than authorized request sender (now they have to be the same user, in future there can be admins, ...)